apps/elf-council-frontend/src/ui/gsc/useKickGSC.ts

```diff
@@ -0,0 +1,36 @@
+import { useCallback } from "react";
+
+import {
+  useSmartContractTransaction,
+  UseSmartContractTransactionOptions,
+} from "@elementfi/react-query-typechain";
+import { Signer } from "ethers";
+
+import { gscVaultContract } from "src/elf/contracts";
+import { GSCVault } from "@elementfi/elf-council-typechain";
+
+const EMPTY_BYTE = "0x00";
+
+export function useKick(
```

Comment:
`useKick` should just return the useSmartContractTransactionEnvelope.  Leave it to the consumer to create a handleLeave method.  I would make a separate `useKickCallback` function or something that could return a handleLeave method.  That could probably just live in the component file or separate file altogether.